src/for-redux-swr/SpreadoSetupImpl.ts

```diff
@@ -0,0 +1,41 @@
+/* eslint react-hooks/rules-of-hooks: "off" */
```

Comment:
nit: Perhaps we may use `// eslint-disable-next-line` just like in `for-redux-react-query/SpreadoSetupImpl.ts`?